frontend/src/views/Pages/UserProfileTransactionList.spec.js

```diff
@@ -30,8 +20,12 @@ describe('UserProfileTransactionList', () => {
     },
   }
 
+  const stubs = {
```

Comment:
Does this make that the underlying component is mocked!?

Reply:
It does not load the stubbed component. Instead it places a stub, on which you can emit events.